app/src/main/java/com/example/weatherapp/models/LocationWeatherData.kt

```diff
@@ -0,0 +1,6 @@
+package com.example.weatherapp.models
+
+data class LocationWeatherData(
+    val address: String?,
+    val days: List<TemperatureData>?,
```

Comment:
Similar comment to the `Location` class. Here we are sharing `TemperatureData` with both `LocationWeatherData` and `LocationWeatherRemoteData`. To properly segregate data models from domain models, we shouldn't share things between them

Reply:
Changed the name from 'TemperatureData' to 'DaysTemperatureData'